corona-ide-main/src/main/java/com/coronaide/main/config/UIControllerConfiguration.java

```diff
@@ -0,0 +1,23 @@
+package com.coronaide.main.config;
```

Comment:
Corona Main was meant more as the entry point - should decide long-term if the UI should live here or it's own project. For now, lets get it working and see how it evolves